astrokat/simulate.py

```diff
@@ -154,9 +154,9 @@ def __init__(self, kat, **kwargs):
         self.start_time = datetime2timestamp(simobserver.date.datetime())
         if "durations" in self.obs_params:
             if "start_time" in self.obs_params["durations"]:
-                self.start_time = datetime2timestamp(
-                    self.obs_params["durations"]["start_time"]
-                )
+                self.start_time = katpoint.Timestamp(
+                    str(self.obs_params["durations"]["start_time"])
+                ).secs
```

Comment:
Sorry, I know I mentioned changing this in a previous comment, but I suggest we don't change it.  When I was working on #103, I saw that `datetime2timestamp` and `timestamp2datetime` as used consistently across astrokat, so better to leave it that way.  In #103 I added tests that ensure `katpoint.Timestamp` conversion agrees with the astrokat functions instead.

Reply:
Not a problem at all, I rolled back the `katpoint.Timestamp` updates to the original `datetime2timestamp` implementation